tests/drop_counters/test_drop_counters.py

```diff
@@ -0,0 +1,383 @@
+import pytest
+import ptf.testutils as testutils
+import logging
+import pprint
+import random
+import time
+import scapy
+import yaml
+import re
+import os
+import json
+
+pytestmark = [
+    pytest.mark.disable_loganalyzer  # disable automatic loganalyzer
+]
+
+logger = logging.getLogger(__name__)
+
+IP_DST = "10.0.0.59"
```

Comment:
This is hard coded, while can be retrieved from minigraph_facts

Reply:
Fixed.